src/digital.cpp

```diff
@@ -0,0 +1,47 @@
+#include "digital.h"
+#include <Arduino.h>
+
+#include "STM32TimerInterrupt_Generic.h"
+// #include "const.h"
+
+// Ticker to poll input readings at fixed rate
+STM32TimerInterrupt dataReader(TIM15);
+
+void initDigital(){
+    //outputs
```

Comment:
These comments are a bit superfluous since we can see in the pinMode call that they are outputs or inputs.